salt/pkgrepo/ubuntu/init.sls

```diff
@@ -1,3 +1,16 @@
+legacy-saltstack-pkgrepo:
```

Comment:
If there is a common syntax pattern to make this removal opt-in, that's much better to avoid breaking people's workflows if they've pinned an older release.